source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/row/products/product-item.tsx

```diff
@@ -23,15 +25,15 @@ type IProps =
 
 class ProductItem extends React.Component<IProps> {
   getActiveProductColumns = () => {
-    const { product, moment, timezone, activeProductColumns } = this.props;
+    const { product, moment, timezone, activeProductColumns, productDefinition } = this.props;
 
     const {
-      name,
```

Comment:
a product does not have a name, but the product definition does

Reply:
also has a description and an applicationType